packages/shipjs/src/step/prepare/__tests__/updateChangelog.spec.js

```diff
@@ -1,24 +1,26 @@
 const path = require('path');
 import { prepareParams } from '../updateChangelog';
+import { parseArgs } from '../../../util';
 import tempWrite from 'temp-write';
 
 describe('prepareParams', () => {
   it('loads configuration from --config option', () => {
+    parseArgs.mockImplementation(jest.requireActual('../../../util').parseArgs);
     const config = {
       writerOpts: {
         headerPartial: '## {{version}}',
       },
     };
-    const configString = `module.exports = { ${config} };`;
+    const configString = `module.exports = ${JSON.stringify(config)};`;
     const configPath = tempWrite.sync(configString);
     const configDir = path.basename(path.dirname(configPath));
 
     const { args } = prepareParams({
       dir: configDir,
-      conventionalChangelogArgs: `--config ${configPath}`,
+      conventionalChangelogArgs: `-i CHANGELOG.md -s --config ${configPath}`,
```

Comment:
I noticed you added the `-i`/`--infile` option so that there is not the `undefined` error down the line with this flag. Too late for this PR as you already closed it, but maybe things could be improved here: arguments parsing shouldn't yield an error if a flag is missing, no?

Reply:
You're right.

https://github.com/algolia/shipjs/blob/06c7226fa66185edd23f6626b6a2b8b7df1d93b6/packages/shipjs/src/step/prepare/updateChangelog.js#L100-L101

We have these two lines expecting to have `infile` and `outfile` inside the parsed args. So the correct way to fix it would be to throw a better error message to dev and to document that these are mandatory.